hpcgap/lib/oper.g

```diff
@@ -1805,6 +1805,10 @@ BIND_GLOBAL( "InstallGlobalFunction", function( arg )
         func := arg[2];
     fi;
     if IS_STRING( oper ) then
+      # Can't use IsBoundGlobal yet in early loading
```

Comment:
this comment feels a bit out of place. I mean, we don't have suchs comments for  `LEN_LIST`, `IS_STRING` or `VALUE_GLOBAL`...